.php_cs.dist

```diff
@@ -102,9 +102,8 @@ return Config::create()
             'location' => 'after_open',
             'separate' => 'bottom',
         ],
-        'native_function_invocation' => [
-            'include' => ['@compiler_optimized'],
-        ],
+        'native_function_invocation' => false,
```

Comment:
Just as an idea: in my experience working with legacy projects I have found it a lot easier to enable fixers one-by-one:

- push a commit enabling a fixer, usually breaking the build
- push a commit applying the fixer

This has the following advantages:

- pull requests are easier to review
- discussions are only related to enabling a single fixer, not a bunch of them

What do you think?

Reply:
Of course it's easier, but it requires me to open 10 PRs each conflicting :)

Reply:
Also note that I kinda capture them in each commit. Some commits may include multiple ones but it means it had no effect